src/templates/layouts/pressRelease/index.tsx

```diff
@@ -0,0 +1,155 @@
+                        <p className="vads-u-margin-top--1 vads-u-margin-bottom--0">
+                          {contact.title}{' '}
+                          {contact.field_description &&
+                            `, ${contact.field_description}`}
+                        </p>
+                        <p className="vads-u-margin-top--1 vads-u-margin-bottom--0">
+                          {contact.field_phone_number}
+                        </p>
+                        {contact.field_email_address && (
+                          <p className="vads-u-margin-top--1 vads-u-margin-bottom--0">
+                            <a href={`mailto:${contact.field_email_address}`}>
+                              {contact.field_email_address}
+                            </a>
+                          </p>
+                        )}
+                      </div>
+                    )
+                  })}
+                </section>
+                {downloads.length > 0 && (
+                  <section className="vads-u-margin-bottom--6">
+                    <div className="vads-u-font-weight--bold vades-u-margin-bottom--1">
+                      Download media assets
+                    </div>
+                    {/*Print out unorder list links per type*/}
+                    {downloads.map((asset) => {
+                      if (!asset) {
+                        return null
+                      }
+                      return (
+                        <ul key={asset.id}>
+                          <li>
+                            {asset.type === 'DrupalMediaDocument' && (
+                              <a href={asset.field_document.uri.url} download>
+                                {asset.name}
+                              </a>
+                            )}
+                            {asset.type === 'DrupalMediaImage' && (
+                              <a href={asset.image.uri.url} download>
+                                {asset.name}
+                              </a>
+                            )}
+                            {asset.type === 'DrupalMediaVideo' && (
+                              <a href={asset.field_document.uri.url} download>
+                                {asset.name}
+                              </a>
+                            )}
+                          </li>
+                        </ul>
+                      )
+                    })}
+                  </section>
+                )}
+
+                <section className="vads-u-margin-bottom--6 vads-u-text-align--center">
+                  {/* ### */}
+                </section>
+
+                <section className="vads-u-margin-bottom--3">
+
+                  {/* should be fieldOffice.entity.fieldPressReleaseBlurb.processed */}
+                </section>
+                <a
+                  onClick={() =>
+                    recordEvent({ event: 'nav-secondary-button-click' })
+                  }
+                  href="#"
```

Comment:
`href={listing}` if listing is included in the formatter (`listing: entity.field_listing?.path.alias,`)